swan/python/configure_kernels_and_terminal.py

```diff
@@ -0,0 +1,73 @@
+"""
+Writes kernel.json and terminal .bash_profile environment variables
+from the current process environment
+"""
+
+import json
+import logging
+import os
+import re
+import shutil
+
+SWAN_ENV_FILE = os.environ["SWAN_ENV_FILE"]
+KERNEL_DIR = os.environ["KERNEL_DIR"]
+HOME = os.environ["HOME"]
+PYTHONEXECPATH = shutil.which(f"python3")
+
+
+def modify_python_argv(kernel_json):
+    """Make sure that `python` points to the correct binary from CVMFS"""
+    if re.match(r"python[0-9.]*", kernel_json["argv"][0]):
+        kernel_json["argv"][0] = PYTHONEXECPATH
+
+
+def configure_kernels():
+    """Writes kernel.json with env and argv from software stack"""
+    kernel_json_file_names = [
+        f"{KERNEL_DIR}/{kernel_dir}/kernel.json"
+        for kernel_dir in os.listdir(KERNEL_DIR)
+    ]
+    for kernel_json_file in kernel_json_file_names:
+        kernel_json = None
+        with open(kernel_json_file, "r") as file:
+            kernel_json = json.loads(file.read())
+
+        kernel_json["env"] = dict(os.environ)
```

Comment:
should we merge envs instead of replacing?

Reply:
I added a condition for that now